src/composable/useSearchDeveloper.js

```diff
@@ -0,0 +1,31 @@
+import sortBy from "lodash/sortBy";
+import Searcher from "@/util/Searcher";
+import store from "../store/store";
+
+const developerSearcher = new Searcher().setField("name").setField("login");
+
+export function useSearchDeveloper() {
+  function searchPopular(searchTerm) {
+    const developers = store.state.Developers.developers;
+    developerSearcher.setData(developers);
+
+    if (!searchTerm) {
+      return developers.slice(0, 10);
+    }
+
+    return sortBy(developerSearcher.findAll(searchTerm), dev => -dev.followers).slice(0, 10);
```

Comment:
Have you considered having a single return point here only? LIke just using the conditional to decide whether to use the initial `developers` value or the filtered one?